magic-link/magic-link-templates.php

```diff
@@ -295,7 +295,7 @@ private function localized_selected_field_settings( $template ) {
         if ( !empty( $template['fields'] ) ) {
 
             $localized_selected_field_settings = [];
-            foreach ( $template['fields'] ?? [] as $template_field ) {
+            foreach ( $template['fields'] as $template_field ) {
```

Comment:
@kodinkat, feels like that `?? []` should be kept. No?

Reply:
Empty if statement check on ln:295 confirms fields array exists...

Reply:
👍